### PR TITLE
sci-geosciences/bt747: update EAPI 6 -> 8

### PR DESCRIPTION
**Pull request by vaukai**
@wraeth

### CODE REVIEW

File: dev-java/commons-imaging/Manifest [hunk 1]
```diff
@@ -1,2 +1,7 @@
...
 DIST commons-imaging-1.0-alpha3-src.tar.gz.asc 833 BLAKE2B b28631badf344fa476a99db78e26a9453d033d21dba8484b2621a4cb0295ea262f58d2df4f8afd6d8090ecd0ea98ec2dca0168e2b401d662c335ad1ef9518b98 SHA512 b7e6691d5eac6f81657dc42aa69184da7fd1bf364da391ac4e1a27991d9a8bbba23296d58b874b4c69a20d39c3c1e821ca19c312f3e71b3acd8903156a2a0a1d
+=======
+DIST apache-sanselan-incubating-0.97-src.tar.gz 33317875 BLAKE2B c9db6c30accc308d2c5d5977b84da118ab50b010b3728665a75dab13f9fecb70075e2d2208a62fabc57af7396f7c39b1ec498e0840e5a2c9c8de9b95d2dd597a SHA512 b7e8af01f4b4ca812313bbf573758d9c0e1355266bfb83898336b14c4b492a8979f2a593f00d53c076e83cf807a049df624491a26623cfbceb00560b748f05f2
+DIST commons-imaging-1.0-alpha2-src.tar.gz 39798631 BLAKE2B 01d0754689896a1f2ff1401cac7d9f59a6bbb245c0880f24f857efc53975c1203c6857d7a5d107763741a2dec7fb48c4faa258786f574cf2f3c5acca251e368b SHA512 f2479e0dc726a8d1193f4eef6289efe493cfbbe257054b5a13e929f5785f7a2c9c937e3f379e0f0066c591d18be27d12f1e5157e789eebab78fba489cc81e688
+>>>>>>> parent of b28ee9959f57 (dev-java/commons-imaging: Drop 0.97)
```
Comment (fordfrog) [inline dev-java/commons-imaging/Manifest]: this should be resolved, that conflict

Comment (fordfrog) [reply dev-java/commons-imaging/Manifest]: ah, it's fixed in the following commit...

Comment (fordfrog) [reply dev-java/commons-imaging/Manifest]: ok, so it has to be resolved in this commit and not in the following one, as this broken manifest cannot be pushed to the git repo, the hook rejects it.

Comment (vaukai) [reply dev-java/commons-imaging/Manifest]: tried my best, but it was a complicated operation. so maybe still not sufficient. if running into trouble with EAPI5 in the first commit, we could squash second commit into first (that's what i did with jcommander:1.64).

**Comment by arthurzam** [comment]
@vaukai Merged the bt747 stuff

**Pull request closed without merging**